sherlockml/clients/user.py

```diff
@@ -14,38 +14,70 @@
 
 
 from collections import namedtuple
+from enum import Enum
 
 from marshmallow import Schema, fields, post_load
+from marshmallow_enum import EnumField
 
 from sherlockml.clients.base import BaseClient
 
 
-User = namedtuple("User", ["id"])
+class GlobalRole(Enum):
+    BASIC_USER = "global-basic-user"
+    FULL_USER = "global-full-user"
+    ADMIN = "global-admin"
+
+
+User = namedtuple(
+    "User",
+    [
+        "id",
+        "username",
+        "full_name",
+        "email",
+        "created_at",
+        "enabled",
+        "global_roles",
+    ],
+)
 
 
 class UserSchema(Schema):
+
     id = fields.UUID(data_key="userId", required=True)
+    username = fields.Str(required=True)
+    full_name = fields.Str(data_key="fullName", required=True)
+    email = fields.Str(required=True)
+    created_at = fields.DateTime(data_key="createdAt", required=True)
+    enabled = fields.Boolean(required=True)
+    global_roles = fields.List(
```

Comment:
I believe our existing convention is to use `many=True` on the inner item's schema rather than putting it in a `fields.List`. I like the explicitness of the latter, but the former is more compact. Should we adopt a consistent style here?